lib/commands/connect/import.js

```diff
@@ -15,21 +17,14 @@ module.exports = {
     ],
     needsApp: true,
     needsAuth: true,
-    run: function (context) {
-      api.withUserConnections(context.auth.password, context.app, context.flags).then(function(connections){
-          if (connections.length > 1) {
-            console.log("Multiple connections found. Please use '-r' to specify a single connection by resource name.");
-          } else {
-            let connection = connections[0];
-            let url = 'https://' + api.HOST + ':443' + '/api/v3/connections/' + connection.id + '/actions/import';
-            fs.readFile(context.args.file, 'utf8', function(err, data) {
-              data = JSON.parse(data);
-              api.request(context.auth.password, 'POST', url, data).then(function(response) {
-                console.log("Upload complete");
-                console.log(response.text);
-              });
-            });
-          }
-      });
-    }
+    run: cli.command(co.wrap(function* (context, heroku) {
+      let connection = yield api.withConnection(context, heroku);
+      let url = '/api/v3/connections/' + connection.id + '/actions/import';
+      fs.readFile(context.args.file, 'utf8', co.wrap(function* (err, data) {
+        data = JSON.parse(data);
```

Comment:
Right now, they get a raw JavaScript exception with details:

```console
$ heroku connect:import protected-ridge-4963-herokuconnect-amorphous-4711.json
undefined:22
    "connection": {
    ^

SyntaxError: Unexpected string
    at Object.parse (native)
    at /Users/malchin/apps/heroku-connect-plugin/lib/commands/connect/import.js:26:27
    at tryToString (fs.js:414:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:401:12)
```

We *might* be able to wrap it in something a bit more useful, but for now, it at least shows enough information for them to be able to resolve it, I think.